{"name":"C - Exception Handling","group":"AtCoder - AtCoder Beginner Contest 134","url":"https://atcoder.jp/contests/abc134/tasks/abc134_c","interactive":false,"memoryLimit":1024,"timeLimit":2000,"tests":[{"input":"3\n1\n4\n3\n","output":"4\n3\n4\n","id":1699991922772},{"id":1699991922768,"input":"2\n5\n5","output":"5\n5"}],"testType":"single","input":{"type":"stdin"},"output":{"type":"stdout"},"languages":{"java":{"mainClass":"Main","taskClass":"CExceptionHandling"}},"batch":{"id":"906de3ba-1b24-4eb7-a57f-7f4541eeb821","size":1},"srcPath":"/Users/nitishkumar/Documents/Template_Codes/Python/CP/Codeforces/C_Exception_Handling.py"}